spec/services/saml_request_validator_spec.rb

```diff
@@ -59,6 +72,17 @@
       end
     end
 
+    context 'no authn context and sp and authorized nameID format' do
```

Comment:
Nitpick: the tests underneath this one all say "and valid sp" which seems a little more specific

```suggestion
    context 'no authn context and valid sp and authorized nameID format' do
```